Source/WebKit/NetworkProcess/NetworkProcess.h

```diff
@@ -271,7 +271,7 @@ class NetworkProcess : public AuxiliaryProcess, private DownloadManager::Client,
     void setTopFrameUniqueRedirectTo(PAL::SessionID, TopFrameDomain&&, RedirectedToDomain&&, CompletionHandler<void()>&&);
     void setTopFrameUniqueRedirectFrom(PAL::SessionID, TopFrameDomain&&, RedirectedFromDomain&&, CompletionHandler<void()>&&);
     void registrableDomainsWithWebsiteData(PAL::SessionID, OptionSet<WebsiteDataType>, bool shouldNotifyPage, CompletionHandler<void(HashSet<RegistrableDomain>&&)>&&);
-    void didCommitCrossSiteLoadWithDataTransfer(PAL::SessionID, RegistrableDomain&& fromDomain, RegistrableDomain&& toDomain, OptionSet<WebCore::CrossSiteNavigationDataTransfer::Flag>, WebPageProxyIdentifier, WebCore::PageIdentifier);
+    void didCommitCrossSiteLoadWithDataTransfer(PAL::SessionID, RegistrableDomain&& fromDomain, RegistrableDomain&& toDomain, OptionSet<WebCore::CrossSiteNavigationDataTransfer::Flag>, WebPageProxyIdentifier, WebCore::PageIdentifier, bool didFilterLinkDecoration);
```

Comment:
Please make this a boolean enum DidFilterLinkDecoration { Yes, No } so that it's readable on the caller side too. Unless it for some reason has to be a primitive type. I would also prefer a more precise name such as DidFilterKnownLinkDecoration if it indeed is a case of known keys on a list.